crowdsec-docs/unversioned/getting_started/post_installation/health_check.mdx

```diff
@@ -0,0 +1,335 @@
+
+  Check CAPI status:
+  <CodeBlock className="language-bash">sudo cscli capi status</CodeBlock>
+  - Should show: 
+    - `INFO You can successfully interact with Central API (CAPI)`
+    - Along with information about the connectivity config file path and if your Security engine is enrolled in CrowdSec console.
+
+  Common issues include:
+  - Missing `online_api_credentials.yaml` in your CrowdSec config directory
+    - If they don't exist, you can create them by running the command:
+      <CodeBlock className="language-bash">sudo cscli capi register</CodeBlock>
+  - Firewall rules blocking outbound connections to the CrowdSec Central API (api.crowdsec.net)
+  - DNS resolution issues.
+  - Proxy server configuration.
+  - Connectivity issues within Docker containers.
+
+</details>
+
+## ✋🏻 Remediation checks
+
+### *Validate Blocks or Captchas* 
+
+Now that detection and connectivity are working, let’s verify that your bouncers are correctly applying remediation on malicious IPs.  
+
+**Prerequisite:**  
+To apply remediation with CrowdSec, you’ll need a bouncer — available for firewalls, web servers (like Nginx), reverse proxies, CDNs, cloud WAFs, edge appliances, [and more](https://doc.crowdsec.net/u/bouncers/intro).  
+
+<details>
+  <summary>✋🏻 Bouncer Remediation test</summary>
+
+This test involves manually creating a **decision** against a public IP of one of your devices for a very short period (1 minute).  
+
+:::danger 
+BE CAREFUL -- Risk of Self-Lockout  
+This procedure will temporarily block your access to the services protected by your bouncer.  
+Make sure to properly follow the instructions to set the TTL to a low expiration time (1 minute). 
+OR do it from a device with a different public IP address than the client you're using to setup CrowdSec.
+:::
+
+1️⃣ Find your public IP:
+<CodeBlock className="language-bash">curl api.ipify.org</CodeBlock>
+or
+<CodeBlock className="language-bash">curl curl ipinfo.io/ip</CodeBlock>
+
+2️⃣ Add a ban decision for your IP (valid for 1 minute):
+<CodeBlock className="language-bash">sudo cscli decisions add ban --ip \<your-public-ip\> --duration 1m --reason "CrowdSec remediation test"</CodeBlock>
+
+3️⃣ Try accessing your service (e.g. website, API). from the same public IP address. 
+  ➡️ You should be blocked by the bouncer. returning a forbidden response (HTTP 403) or a captcha challenge.
+
+4️⃣ Wait for 1 minute, then check the decisions list to see if the decision has been removed
+
+</details>
+
+### Were all the tests successful ?
+
+If you were successfully blocked, congratulations! Your remediation setup is working correctly. 🎉  
+
+You might want to continue to the next recommended steps:
+- Enroll your Security Engine to the [CrowdSec Console](/u/getting_started/post_installation/console/)
+- Then subscribe to more blocklists to benefit from additional proactive prevention
+
+<details>
+    <summary>🐞 **Remediation Troubleshooting**</summary>
```

Comment:
Remediation troubleshooting needs to be at the same level as the detection troubleshooting with separate details for each major mentioned points